pkg/anago/release.go

```diff
@@ -445,4 +454,50 @@ func (d *DefaultRelease) CreateAnnouncement() error {
 	return nil
 }
 
+// UpdateGitHubPage Update the GitHub release page, uploading the
+// source code
+func (d *DefaultRelease) UpdateGitHubPage() error {
+	// Array of assets to be published in the release page
+	assetList := []string{
+		// Build the path to the kubernetes tar file:
+		filepath.Join(
+			gitRoot, // /workspace/src/k8s.io/kubernetes
+			fmt.Sprintf("%s-%s", release.BuildDir, d.state.versions.Prime()), // _output-v1.20.0-beta.3/
+			release.GCSStagePath,     // gcs-stage/
+			d.state.versions.Prime(), // v1.20.0-beta.3
+			release.KubernetesTar,    // kubernetes.tar.gz
+		) + ":Kubernetes Source Code",
+	}
+
+	// URL to the changelog:
+	changelogURL := fmt.Sprintf(
+		"https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-%d.%d.md",
+		d.state.semverBuildVersion.Major,
+		d.state.semverBuildVersion.Minor,
+	)
+
+	// Build the options set for the GitHub page
+	ghPageOpts := &announce.GitHubPageOptions{
+		AssetFiles:            assetList,
+		Tag:                   d.state.versions.Prime(),
+		NoMock:                d.options.NoMock,
+		UpdateIfReleaseExists: true,
+		Name:                  "Kubernetes " + d.state.versions.Prime(),
+		Draft:                 false,
+		Owner:                 "kubernetes",
+		Repo:                  "kubernetes",
```

Comment:
```suggestion
		Owner:                 git.DefaultGithubOrg,
		Repo:                  git.DefaultGithubRepo,
```

Reply:
:white_check_mark: done.

Do you think that later we should have these values in anago or maybe in release?

While I was considering using them yesterday it seemed to me that while having them in git seems like a good failsafe/sane default, if we see the git package in as more general purpose tool perhaps the release process should have it's own constants, decoupled from the git pkg default.

Reply:
I think I did the same thing in the release notes source btw ;) 
I'll fix it at some point.